docs/Lifecycle.md

```diff
@@ -2,7 +2,7 @@
 
 ## Lifecycle
 Following the schema of the internal lifecycle of Fastify.<br>
-On the right branch of every section there is the next phase of the lifecycle, on the left branch there is the corresponding error code that will be generated if the parent throws an error *(note that all the errors are automatically handled by Fastify)*.
+On the right branch of every section, there is the next phase of the lifecycle, on the left branch there is the corresponding error code that will be generated if the parent throws an error *(note that all the errors are automatically handled by Fastify)*.
```

Comment:
This sentence doesn't make sense to me with the new comma. The two parts it splits seem to be one in my reading.

Reply:
On second glance you are correct, will remove!